src/components/sequencing/Sequences.svelte

```diff
@@ -59,6 +63,41 @@
     const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
     goto(`${base}/sequencing/new${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}`);
   }
+
+  async function importLibarySequences(): Promise<void> {
+    const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
+    if (workspaceId === null) {
+      console.log("Workspace doesn't exist");
+      showFailureToast("Library Import: Workspace doesn't exist");
+      return;
+    }
+    const { confirm, value } = await showLibrarySequenceModel();
+
+    if (confirm && value) {
+      const fileName = await value.libraryFile.name;
+      const type = fileName.slice(fileName.lastIndexOf('.') + 1);
+      if (type !== 'satf') {
```

Comment:
Are files that contain satf code, but don't have the extension on it allowed? If the file extension is required, then you can specify valid file extensions to disallow unsupported files from even being selected also.

Reply:
Good point. I add `accept=".satf"` to only allow for satfs to be imported.